core/global_constants.cpp

```diff
@@ -88,7 +88,8 @@ static Vector<_GlobalConstant> _global_constants;
 VARIANT_ENUM_CAST(KeyList);
 VARIANT_ENUM_CAST(KeyModifierMask);
 VARIANT_ENUM_CAST(ButtonList);
-VARIANT_ENUM_CAST(JoystickList);
+VARIANT_ENUM_CAST(SDLJoyButton);
+VARIANT_ENUM_CAST(SDLJoyAxis);
```

Comment:
What's the rationale for renaming those with `SDL` prefix?

It will break compatibility and IMO it's not particularly relevant for the end users that the format we use is the same as SDL's - we don't actually use any SDL code beyond their mappings.

Reply:
None. I'll drop the SDL.